xep-0045.xml

```diff
@@ -1385,6 +1385,7 @@
 </presence>
 ]]></example>
       <p>Before attempting to enter the room, a MUC-compliant client SHOULD first discover its reserved room nickname (if any) by following the protocol defined in the <link url='#reservednick'>Discovering Reserved Room Nickname</link> section of this document.</p>
+      <p>When a MUC service receives an &lt;x/&gt; tagged join stanza from an already-joined client (as identified by the client's full JID), the service SHOULD assume that the client lost its synchronization, and therefore it SHOULD send exactly the same stanzas to the client as if it would actually join the MUC. The server MAY also send a presence update to the other participants according to the received join presence.</p>
```

Comment:
> SHOULD assume that the client lost its synchronization, and therefore it

nit: I'd strike this and just say "the service SHOULD send…"

> send exactly the same stanzas to the client as if it would actually join the MUC
 
I think this needs a reference to what those stanzas are "(See section x.x)". Also,  minor nit, "as if it would actually join the MUC" needs to be rephrased, it doesn't sound right.